Fix wrong Postgres search_path set up instructions

tdnguyen6 opened the pull request:
In the set up instructions for Postgres database backend, the instructed command for setting search_path to the correct schema lacks `public` and `utility`, which prevented SqlAlchemy to find all those required schemas.

tdnguyen6 committed:
Fix wrong Postgres search_path set up instructions

In the set up instructions for Postgres database backend, the instructed command for setting search_path to the correct schema lacks `public` and `utility`, which prevented SqlAlchemy to find all those required schemas.

xinbinhuang commented on docs/apache-airflow/howto/set-up-database.rst:
```diff
@@ -214,7 +214,7 @@ We recommend using the ``psycopg2`` driver and specifying it in your SqlAlchemy
 
 Also note that since SqlAlchemy does not expose a way to target a specific schema in the database URI, you may
-want to set a default schema for your role with a SQL statement similar to ``ALTER ROLE username SET search_path = airflow, foobar;``
+want to set a default schema for your role with a SQL statement similar to ``ALTER ROLE airflow_user SET search_path = airflow, public, utility;``
```
```suggestion
want to set a default schema for your user with a SQL statement similar to ``ALTER USER airflow_user SET search_path = airflow, public, utility;``
```

`ALTER ROLE` is alias to `ALTER USER`, so I think it'll be more consistent with the previous sections (i.e. `CREATE UESER ...`).

tdnguyen6 committed:
change ALTER ROLE to ALTER USER to be consistent with previous sections

Co-authored-by: Xinbin Huang <bin.huangxb@gmail.com>

tdnguyen6 committed:
Clarify search_path instructions for Postgres

Postgres config: 
The step of configuring the search_path is only necessary when Postgres user's  search_path has been configured to not include public as all airflow tables lie in public schema.
The default search_path is "$user", public, which already contains pubic.
